src/Symfony/Bundle/WebProfilerBundle/EventListener/WebDebugToolbarListener.php

```diff
@@ -25,8 +25,7 @@
  *
  * The onKernelResponse method must be connected to the kernel.response event.
  *
- * The WDT is only injected on well-formed HTML (with a proper </body> tag).
- * This means that the WDT is never included in sub-requests or ESI requests.
+ * The WDT is only injected on well-formed HTML (with a proper <title> tag).
```

Comment:
I thought this change has been rejected in the previous PR (see https://github.com/symfony/symfony/pull/20343#issuecomment-257884538, no matter of the branch it targets), the WDT should never be injected in an ESI request.